### PR TITLE
main/cryptsetup: update to 2.7.1, main/cryptsetup-scripts: update to 2.7.0.1

### PR DESCRIPTION
**Pull request by mhmdanas**

### CODE REVIEW

File: main/cryptsetup-scripts/update.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-pattern = r"cryptsetup\ Debian\ release\ 2:([\d.\-~rc]+)"
-ignore = ["*rc*", "*~*"]
+pattern = r"cryptsetup\ Debian\ release\ 2:([\d.\-~rc\+exp]+)"
```
Comment (mhmdanas) [inline main/cryptsetup-scripts/update.py:1]: How about we match all letters maybe, or even just match the version in URLs like https://salsa.debian.org/cryptsetup-team/cryptsetup/-/tags/debian%2F2%252.7.1-1+exp?

Comment (nekopsykose) [reply main/cryptsetup-scripts/update.py:1]: hm not sure what the benefit would be but maybe i'm too tired. does the existing check fail in some case?

Comment (mhmdanas) [reply main/cryptsetup-scripts/update.py:1]: If you mean the original, it matches experimental versions (like the version in the link above).

Comment (nekopsykose) [reply main/cryptsetup-scripts/update.py:1]: hm seems ok then, sure

**Pull request merged**